mergify_engine/tests/functional/actions/test_post_check.py

```diff
@@ -128,8 +128,7 @@ def test_checks_feature_disabled(self):
             if self.SUBSCRIPTION_ACTIVE
             else frozenset(),
         )
-        loop = asyncio.get_event_loop()
-        loop.run_until_complete(self.subscription.save_subscription_to_cache())
+        utils.async_run(self.subscription.save_subscription_to_cache())
```

Comment:
```suggestion
        asyncio.run(self.subscription.save_subscription_to_cache())
```
should work since there's no `gather()` involved.